src/click/core.py

```diff
@@ -2537,6 +2527,11 @@ def consume_value(self, ctx, opts):
 
         return value, source
 
+    def process_value(self, ctx, value):
```

Comment:
Looks like this got added by mistake when rebasing. This shouldn't exist anymore.

Reply:
Ohh, right. My bad. I'll remove that.